### DISCUSSION TITLE
where required packages are stored?

### DISCUSSION THREAD

**Discussion by gguio** [pyodide/pyodide#5022] [Q&A]
I want to use pyodide npm package and in readme it says that I need to download and extract pyodide-build-*.tar.bz2, but I cannot find such an archive in realeses. There are pyodide-*.tar.bz2, pyodide-core-*.tar.bz2 and others... So which one I need?

**Comment by hoodmane** [comment]
I think that is out of date. Try just calling `pyodide.loadPackage("whatever-package")` and if it's in Pyodide's package set, Pyodide will download it automatically.

**Comment by ryanking13** [comment]
It will download the package on-the-fly as hoodmane mentioned, but you can also download the archive from the GitHub releases: https://github.com/pyodide/pyodide/releases/tag/0.26.2
